### ISSUE TITLE
do not show warnings if everything was successfully imported

### ISSUE THREAD

**Issue by benben** [contentful/contentful-import#76]
After a successful import, it still prints all the rate limit warnings, which always make me scroll up just to see that despite these warnings, everything went ok. Maybe we can avoid showing these?

**Comment by axe312ger** [comment]
Yes, we are about to group these errors so only one single line will be shown. Should help.

We want to rework the parallelism of the tool, also. Should reduce the warnings, also.

**Comment by ChaosSteffen** [comment]
Question: Why does your own tool run into rate-limiting "errors" when consuming your API? I would consider this the main issue here.
